com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Input/HDRP/HDSceneColorNode.cs

```diff
@@ -49,6 +49,10 @@ internal class HDSceneColorNode : IStandardNode
                     name: "Lod",
                     tooltip: "The mip level to sample"
                 ),
+                new ParameterUIDescriptor(
```

Comment:
The Exposure static bool wasn't showing up, but it did when I added a ParameterUIDescriptor for it.